bookwyrm/static/js/bookwyrm.js

```diff
@@ -48,6 +48,12 @@ let BookWyrm = new (class {
         document
             .querySelector("#barcode-scanner-modal")
             .addEventListener("open", this.openBarcodeScanner.bind(this));
+
+        document
+            .querySelectorAll('form[name="register"]')
+            .forEach((form) =>
+                form.addEventListener("submit", (e) => this.setPreferredTimezone(e, form))
+            );
```

Comment:
It wasn't immediately obvious to me how the JavaScript side of things is organised, so just tried to immitate what is already there. Happy to move this around wherever it would fit better.

Reply:
It isn't immediately obvious to me either and I'm the one who organized it. This seems like a perfectly good place for it :)